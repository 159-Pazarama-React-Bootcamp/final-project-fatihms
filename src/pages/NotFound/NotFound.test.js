/*eslint-disable */

import React from "react";

import { render, screen } from "@testing-library/react";
import renderer from "react-test-renderer";
import { MemoryRouter as Router } from "react-router-dom";
import userEvent from "@testing-library/user-event";
import "@testing-library/jest-dom/extend-expect";

import NotFound from "./index";

describe("NotFound", () => {
  beforeEach(() => {
    render(
      <Router>
        <NotFound />
      </Router>
    );
  });
  test("NotFound renders correctly", () => {
    const tree = renderer
      .create(
        <Router>
          <NotFound />
        </Router>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
