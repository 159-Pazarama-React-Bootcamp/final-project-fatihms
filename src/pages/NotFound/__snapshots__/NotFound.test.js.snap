// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound NotFound renders correctly 1`] = `
<div
  style={
    Object {
      "alignItems": "center",
      "display": "flex",
      "height": "100vh",
      "justifyContent": "center",
    }
  }
>
  <h1
    style={
      Object {
        "color": "darkred",
        "fontSize": "96px",
      }
    }
  >
    404
  </h1>
</div>
`;
